clients/unshaded/src/main/java/tachyon/client/block/BufferedBlockInStream.java

```diff
@@ -29,13 +30,13 @@
 
 /**
  * Provides a stream API to read a block from Tachyon. An instance extending this class can be
- * obtained by calling {@link TachyonBlockStore#getInStream(long)}. The buffer size of the stream
- * can be set through configuration. Multiple {@link BufferedBlockInStream}s can be opened for a
- * block. This class is not thread safe and should only be used by one thread.
+ * obtained by calling {@link TachyonBlockStore#getInStream}. The buffer size of the stream can be
+ * set through configuration. Multiple {@link BufferedBlockInStream}s can be opened for a block.
  *
  * This class provides the same methods as a Java {@link InputStream} with additional methods from
  * Tachyon Stream interfaces.
  */
+@NotThreadSafe
```

Comment:
FYI, In this PR I added the annotations to classes that talked about thread safety in JavaDoc or were similar and a couple more that belonged to the same equivalence class (e.g. *Clients and *ServiceHandlers)